src/hooks/sourceNodes/createNodeFromEntity/upload/index.js

```diff
@@ -11,5 +11,12 @@ module.exports = function buildUploadNode(
     node.entityPayload = entity.payload;
     node.imgixHost = imgixHost;
     node.digest = entity.path + entity.updatedAt;
+
+    // Gatsby Image CDN fields:
+    node.mimeType = node.entityPayload.attributes.mime_type;
+    node.filename = node.entityPayload.attributes.filename;
+    node.url = node.entityPayload.attributes.url;
+    node.width = node.entityPayload.attributes.width;
+    node.height = node.entityPayload.attributes.height;
```

Comment:
They use imgix so I wonder if we can use placeholderUrl here with `%width%`, `%height%`

Reply:
Good call! This was enough to make it work:
```
node.url = node.entityPayload.attributes.url;
node.placeholderUrl = node.url + `?w=20`;
```

Reply:
Didn't realize there were dynamic url parts with `%width%`. Just pushed up another commit